paper/paper.md

```diff
@@ -0,0 +1,432 @@
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+The majority of our organization's founding graduate students (all women) began
+coding in our undergraduate careers or later.
+We wanted to provide a safe place for local young women to begin coding in high
+school, and develop confidence in themselves and their computational skills
+before college.
+A national organization, Girls Who Code, whose mission and values aligned with
+ours, was founded in 2012.
+Girls Who Code's mission is to close the gender gap in technology
+[@saujani_girls_2015].
+Because of our personal experiences and the paucity of women in our field, we
+began a student organization at the University of Michigan in 2017.
+For the past four academic years we have registered annually as a recognized
+Girls Who Code Club because the national organization provided name recognition,
+curriculum resources, guidance for a Capstone Impact Project, and a framework
+for launching a coding club.
+In 2019 we launched a Data Science Summer Experience which is unaffiliated with
+the national Girls Who Code organization.
+
+The national Girls Who Code organization provides a curriculum that teaches
+website and application development through programming languages like HTML and
+Java.
+However, our biomedical science graduate students generally have limited
+experience with these languages.
+Data Scientist was rated the #1 job in America by Glassdoor in 2016-2019, #3 in
+2020, and #2 in 2021 [@stansell_breaking_2019].
+We believe career exploration in data science will optimally prepare our
+students for careers that provide financial stability. <!-- TODO: address comment from AK: "Do we need to establish a clearer link between data science and the use of Python specifically? The PYPL index, used as a proxy for how popular a language is, has ranked Python as it's number one language (https://pypl.github.io/PYPL.html). I'm not sure if this the best way to illustrate the point, but I do think we need more support for the place of Python in the workplace." -->
+By leveraging the data science expertise of our Club facilitators, we created a
+specialized curriculum focused on computational data science in the Python
+programming language.
+
+Girls Who Code encourages participants to learn programming skills while working
+on an Impact Project website or application throughout the Club
+[@hq_gallery_2021].
+We created an open source Data Science curriculum that teaches the requisite
+Python and statistics skills to complete a Capstone Project, which is a data
+analysis project on a data set of interest.
+Using this curriculum, we employ participatory live coding as used by The
+Carpentries, which is an effective method that engages learners
```

Comment:
Specify how this method engages learners?

Reply:
I think the citations are enough